Code_of_Conduct.md

```diff
@@ -0,0 +1,231 @@
+
+#### Level 1: Minor Violation
+**Examples:** Off-topic discussions, mild unprofessional language, first-time minor spam
+
+**Response:**
+- Private warning from Admin
+- Explanation of the issue
+- Guidance on expected behavior
+- No formal record (unless repeated)
+
+#### Level 2: Moderate Violation
+**Examples:** Repeated Level 1 violations, disrespectful behavior, promotional content after warning, disruptive behavior
+
+**Response:**
+- Formal written warning
+- Required acknowledgment of violation
+- Temporary content removal
+- Incident documented in admin log
+- Potential temporary channel restrictions
+
+#### Level 3: Serious Violation
+**Examples:** Harassment, discriminatory behavior, repeated moderate violations, doxxing, threats
+
+**Response:**
+- Temporary suspension (1-30 days)
+- Formal documentation
+- Required commitment to follow CoC
+- Review before reinstatement
+- May include permanent channel restrictions
+
+#### Level 4: Severe Violation
+**Examples:** Threats of violence, CSAM, severe harassment, coordinated attacks, repeated serious violations
+
+**Response:**
+- Immediate permanent ban
+- No appeal possible (or very limited circumstances)
+- Law enforcement contacted if appropriate
+- Other communities notified if relevant
+
+### Progressive Discipline
+
+For repeated violations:
+1. First offense: Level-appropriate response
+2. Second offense: Escalate one level
+3. Third offense: Temporary or permanent ban
+
+**Note:** Severe violations skip progressive discipline and result in immediate permanent ban.
+
+## Appeals
+
+If you believe an enforcement action was made in error:
+
+1. **Contact:** admin@macadmins.org
+2. **Include:**
+   - Your username
+   - Date of enforcement action
+   - Why you believe it was in error
+   - Any supporting evidence
+3. **Timeline:** Appeals reviewed within 7 days
+4. **Decision:** MAF Board makes final decision on appeals
+
+Appeals are only available for Level 2-3 violations. Level 4 (severe) violations have no appeal except in extraordinary circumstances.
+
+## Admin Responsibilities
+
+Admins are responsible for:
+- Clarifying standards of acceptable behavior
+- Taking appropriate and fair corrective action
+- Removing, editing, or rejecting content that violates this CoC
+- Temporarily or permanently banning any member for behaviors they deem inappropriate, threatening, offensive, or harmful
+- Handling all reports with confidentiality and respect
+- Following Admin Runbooks for consistent enforcement
+
+Admins who do not follow or enforce this Code of Conduct may face temporary or permanent removal from the Admin team.
+
+## Scope of Admin Authority
+
+Admins have the authority to:
+- Remove messages or content
+- Temporarily or permanently restrict channel access
+- Temporarily or permanently suspend or ban members
+- Make decisions about workspace settings and policies
+
+Admin decisions can be appealed to MAF leadership through the process above.
```

Comment:
What if the admin being appealed is part of MAF leadership? Is there a process for members to raise concerns about admin behavior? Can members petition to remove an admin?